README.md

```diff
@@ -163,6 +163,13 @@ The `commentstart` linter can automatically fix comments that do not start with
 
 When the `json` tag is present, and matches the first word of the field comment in all but casing, the linter will suggest that the comment be updated to match the `json` tag.
 
+## DuplicateMarkers
+
+The `duplicatemarkers` linter checks for duplicate markers related with specific type or field.
```

Comment:
Does it check if the marker exists on both a type and field? E.g. if we had maxlength on both the field and the type, that would be problematic